lib/atrium/transaction.rb

```diff
@@ -0,0 +1,50 @@
+module Atrium
+  class Transaction
+    include ::ActiveAttr::Model
+
+    attribute :account_guid
+    attribute :amount
+    attribute :category
+    attribute :check_number
+    attribute :created_at
+    attribute :date
+    attribute :description
+    attribute :guid
+    attribute :is_bill_pay
+    attribute :is_direct_deposit
+    attribute :is_expense
+    attribute :is_fee
+    attribute :is_income
+    attribute :is_overdraft_fee
+    attribute :is_payroll_advance
+    attribute :latitude
+    attribute :longitude
+    attribute :member_guid
+    attribute :memo
+    attribute :merchant_category_code
+    attribute :original_description
+    attribute :posted_at
+    attribute :status
+    attribute :top_level_category
+    attribute :transacted_at
+    attribute :type
+    attribute :updated_at
+    attribute :user_guid
+
+    def self.list(user_guid:)
```

Comment:
It is a feature in ruby 2 called keyword arguements. Here is an article on what it does, but basically you can call the method using a hash with the required arguements in any order.

`Atrium::Transaction.list(:user_guid => "USR-123")` is how you would call this.

https://robots.thoughtbot.com/ruby-2-keyword-arguments